bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageHandler.java

```diff
@@ -53,33 +53,42 @@ public class OpenGarageHandler extends BaseThingHandler {
     private final Logger logger = LoggerFactory.getLogger(OpenGarageHandler.class);
 
     private long refreshInterval;
+    private long transitionTimeSecs;
```

Comment:
Same here, you could use the config object.
Applicable to transitionTimeSecs but also to refreshInterval.